terraform/logical/bi.tf

```diff
@@ -41,33 +41,186 @@ resource "kubernetes_secret" "grafana_config" {
   }
 
   data = {
-    GF_SERVER_ROOT_URL           = local.grafana_url
-    GF_SERVER_SERVE_FROM_SUBPATH = true
-    GF_USERS_DEFAULT_THEME       = "light"
+    GF_SERVER_ROOT_URL                      = local.grafana_url
+    GF_SERVER_SERVE_FROM_SUBPATH            = true
+    GF_USERS_DEFAULT_THEME                  = "light"
+    GF_DATABASE_TYPE                        = "mysql"
+    GF_DATABASE_HOST                        = local.db_master_host
+    GF_DATABASE_USER                        = local.db_master_username
+    GF_DATABASE_PASSWORD                    = local.db_master_password
+    GF_ANALYTICS_REPORTING_ENABLED          = false
+    GF_ANALYTICS_CHECK_FOR_UPDATES          = false
+    GF_METRICS_ENABLED                      = false
+    GF_SECURITY_COOKIE_SECURE               = true
+    GF_SECURITY_DATA_SOURCE_PROXY_WHITELIST = "1.1.1.1:1" #Disable all data source proxying
```

Comment:
Point of curiosity: why this particular string value? Makes me think of Cloudflare's DNS server address. :)

Reply:
It is lol. I just needed an IP address to put in that field. I could've used 192.168.1.1 i guess. Just something that cannot ever resolve to an actual user making a request. There's no way to disable the proxy entirely, all you can do is set a bogus whitelist 🤷